src/rez/rex.py

```diff
@@ -633,6 +634,7 @@ def error(self, value):
     def subprocess(self, args, **subproc_kwargs):
         if self.manager:
             self.target_environ.update(self.manager.environ)
+        self.target_environ = self.adjust_env_for_platform(self.target_environ)
```

Comment:
I would drop the return value, and avoid the unnecessary dict copy.  `adjust_env_for_platform` should just alter self.target_environ in-place, asthe previous code already does

Reply:
fixed!